src/didactUri.ts

```diff
@@ -30,6 +30,7 @@ export class DidactUri {
 	private extFilePath : string | undefined;
 	private text : string | undefined;
 	private user : string | undefined;
+	private number : string | undefined;
```

Comment:
I don't understand. Why it is marked as a line added in this PR?

Reply:
Because this work started from the work @sunix did. It turned out to be supplemental to the startDidact in a column issue and unrelated. But it was useful from a testing standpoint.

Reply:
What is this "number" about?

Reply:
so this PR is about adding a command, it takes a string that should be mapped into a enum, etc.
The PR https://github.com/redhat-developer/vscode-didact/pull/165 I did was about having the ability to call any vscode command that would have a number as argument. That is needed for instance to call the a command like `vscode.open` with the optional argument viewColumn that is a number. But could be used with any vscode command that has a number as argument. So completely unrelated.

Reply:
Aside from this discussion I think that renaming this variable would be good because `number` is also a type in TS/JS.

Reply:
> Aside from this discussion I think that renaming this variable would be good because `number` is also a type in TS/JS.

To me, as long as we keep it simple stupid, it compiles, it runs and the code is clean, we should keep it like it. What is important is what we make available to the user independently from the underlying language. This is like an API or a GUI, we shouldn't choose the name of a parameter according to some constraint related to the language used underneath. Personally I don't see any confusion when reading the code. On the other side, it took me some time to understand what was the the `text` argument for and how to use it when trying to build didact URL as a user. I only understood what it was about by reading the source code: this is bad UX IMHO.